_posts/2018-05-07-A-Machine-Learning-Example-For-Business.md

```diff
@@ -0,0 +1,271 @@
+author: mikaelahonen
+excerpt: A practical example of predictive machine learning. Targeted for business people how want to harness machine learning for business but also understand how it actually works.
+tags:
+- machine learning
+- business
+- predictive analytics
+- data science
+---
+
+## Introduction
+A practical example of predictive machine learning.
+Targeted for business people how want to harness machine learning for business but also understand how it actually works.
+
+The aim is to demonstrate:
+* A concrete example without going too deep in theory.
+* The difference between machine learning and reporting.
+* Benefits to business.
+
+For those who are interested in technical part, the `R` code and data is available at the end of the post.
+
+<blockquote>
+According to a definition
+<b>machine learning means
+that a machine can learn without having explicitly
+programmed for the task</b>.
+This doesn't mean that the program code would evolve by itself, but
+different results are produces in different situations depending on the data
+and the set of rules that are applied.
+</blockquote>
+
+## Machine learning use cases
+Let's quickly see some machine learning use cases.
+
+### #1 Predicting by variables
+For example predicting number of visitors in an event according to
+weather, day of week and ticket price. In the simplest case
+the algorithm might just naively count average number of visitor in the similar cases in the past.
+
+![Predicting by variables](/img/a-machine-learning-example-for-business/visitors.png)
+
+### #2 Clustering
+Customers can be segmented automatically according to
+selected variables. The algorithm might not need any input
+to make the grouping.
+
+![Machine learning clustering example](/img/a-machine-learning-example-for-business/machine-learning-example-clustering.png)
+
+### #3 Image recognition and classification
+Identifying the object from an image. The algorithm
+has to be "trained" with already known cases
+to recognize which kind of pixel and color
+combinations are cats and which are dog.
+
+![Image recognition example](/img/a-machine-learning-example-for-business/image-recognition.png)
+
+## A business problem: Online store deliveries
+In the example, I will go through similar case than the #1: The process
+of predicting by variables as it's very common
+scenario in business analytics. Also many of the
+concepts can be used even in image recognition once the
+image pixels has been converted to tabular data.
+
+Let's imagine that we are running an online shop that
+orders nutrition supplements from a wholesaler and sells the products to consumers.
+
+![Machine learning on supply chain management](/img/a-machine-learning-example-for-business/machine-learning-supply-chain.png)
+
+The online store wants to <b>maximize those wholesale orders that will arrive on time</b>. This increases predictability and that way the satisfaction in end users and sales as well.
```

Comment:
"satisfaction in end users" sounds a bit weird. "the end user satisfaction" maybe?

Reply:
Ok.